config/src/keyassignment.rs

```diff
@@ -477,6 +477,27 @@ pub struct InputSelector {
 
     #[dynamic(default)]
     pub fuzzy: bool,
+
+    #[dynamic(default = "default_num_alphabet")]
+    pub alphabet: String,
+
+    #[dynamic(default = "default_description")]
+    pub description: String,
+
+    #[dynamic(default = "default_fuzzy_description")]
+    pub fuzzy_description: String,
```

Comment:
Hello, I think the word `description` isn't a great here, and `prompt` would better fit

Wdyt?

Reply:
I could go through the code and docs and change it, but I am not sure prompt would be a good idea. I would like to keep the naming similar to what you see in PromptInputLine (see https://wezfurlong.org/wezterm/config/lua/keyassignment/PromptInputLine.html ) and at the same time avoid confusion with this overlay. 

Of course, we could change `description` to `prompt` in PromptInputLine too. What do you think @wez?

Reply:
I think description is fine; prompt doesn't feel like quite the right word anyway, and this is consistent with promptinputline.

Reply:
> prompt doesn't feel like quite the right word anyway

:thinking: huh? My understanding of these words:

- `description` for me is a text that describes something, can be long, and doesn't ask for any action
- `prompt` for me is the text just before a text field input, it's a 'call-to-action' (e.g. ends with a `: `)

And from where I see `description` & `fuzzy_description` being used in the code, `prompt` matches more than `description`.

Or maybe I'm missing some meaning of the english language? (I'm not english native)

Reply:
I think description can be used to describe pretty much anything; also something asking for an action. In this case `description` can be used to describe either what your current InputSelector does, what different keys do or really anything else that you think helps you when using the InputSelector, which I think is better than prompt (it is better to be a bit more general, since it has different use cases). 

The way `description` is currently used in the code is not the only way to use it, it is just what I had in mind for myself.

Reply:
For the different use cases I'd use a separate options, one for the prompt and one for the description, if any.

This is what `fzf` does for example, where both the prompt (named `prompt`) and the description (named `header`) exist.
And since they are separate the layout can be changed so that for example the header is displayed before the prompt line or after (but before the items to select)

Reply:
`prompt` *is* a compatible word, it just feels a bit more "active" or stronger than I think is necessary here.  If I hadn't used description elsewhere I'd be more into it here, but as it stands: I don't have sufficiently strong feelings about it to actively want to change it